nss-evening-cohort-24/hack-client-awareness-campaign
# INTEGRATING LOCAL DB AND POLISHING READ/DELETE

## Description
- Integrated userData.js API calls in post.js so that user-specific rendering could occur on PostCards.
- Created userIdent prop in PostCards.
- Modified endpoint variable in PostData and UserData to point towards localhhost string as opposed to pointing towards Client.js .databaseURL property.

## Related Issue
#11 - Closest to MVP of Read/Delete. 

## Motivation and Context
This implementation is necessary for MVP.

## How Can This Be Tested?
- Pull PR branch to local environment
- Stand web server and start database instance in C#
- Navigate to 'Posts' page
- Ensure that only campaigns that have been created by you have edit/delete buttons. If you haven't created any posts, none of the cards should have the edit/delete button.

## Screenshots (if appropriate):
<img width="826" alt="image" src="https://github.com/nss-evening-cohort-24/hack-client-awareness-campaign/assets/97858910/34cf4cb3-d74c-4eb1-a608-e77730976bc8">


## Types of changes
